src/components/structures/MatrixChat.tsx

```diff
@@ -803,6 +804,9 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                 SettingsStore.setValue("showCookieBar", null, SettingLevel.DEVICE, false);
                 hideAnalyticsToast();
                 break;
+            case 'view_send_to_dialog':
```

Comment:
this should be an enum Action defined like Action.SetRightPanelPhase etc along with a typed payload

Reply:
Yes! I seem to have followed the old code style here